### DISCUSSION TITLE
[Pumpkin Plaza] Is this the reward for beta testers?

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2277] [Ideas]
![image](https://user-images.githubusercontent.com/74167741/220251100-c0f30dad-3bb2-46ed-bb5a-843cb575c47b.png)

![image](https://user-images.githubusercontent.com/74167741/220251001-5da30b64-7c56-4087-8dac-fe02acf5f4ac.png)

![image](https://user-images.githubusercontent.com/74167741/220250968-8a260bad-1a26-4750-8501-a220ada583fb.png)

The Beta Pass owners and devs were not able to foresee this?

Who participated in version 1 and 2 of the beta and lost the "Beta pass" now that  is not necessary anymore, must be very sad to see scenes like this.

Once again we have the feeling of playing a beta game without the beta seal.

**Comment by future2back** [comment]
It would be nice for beta pass owners to give feedback on community moderators, whether they help or hinder in times like these.

This is not possible right? Since the beta pass owners are the community moderators themselves.

**Comment by future2back** [comment]
**_"Over 600 Bumpkins turned up and crashed the party......literally, the party server crashed due to so many players"_**


Are 600 Bumpkins so many players?

How many people try to mint a NFT?


We already went through this in the beta.  Was this really not expected?


There were live streamers, youtubers recording content... This really didn't do well for the project's image and  bad feeling this causes players. insecurity and distrust





**_"This was an experimental side project to get the community together and it clearly needs some improvements!"_**

The user who does not have a Beta Pass should not undergo any experiments.




The question is: Will it always be like this?




![image](https://user-images.githubusercontent.com/74167741/220296469-c82d9764-df05-41fb-afbc-57d43fab1e59.png)

**Comment by adamhannigan** [comment]
Thanks for your feedback @future2back 

We definitely framed this incorrectly and should of been clearer in our communication with users about this feature. The purpose was to get feedback and help load test this experiment.

We load tested this with 200 Bumpkins, not expecting 600+ to show up 😢
